guides/common/modules/con_upgrading-el-on-project-or-proxy.adoc

```diff
@@ -2,12 +2,15 @@
 = Upgrading {EL} on {Project} or {SmartProxy}
 
 {Project} and {SmartProxy} are supported on both {EL} 8 and {EL} 9.
-You can use the following methods to upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9:
-
-* Leapp in-place upgrade
-* Migration by using backup and restore
-* Migration by using cloning
+You can use the following methods to upgrade your {Project} or {SmartProxy} operating system from {EL} 8 to {EL} 9:
 
+Leapp in-place upgrade::
 With Leapp, you can upgrade your {Project} or {SmartProxy} in place therefore it is faster but imposes a downtime on the services.
```

Comment:
```suggestion
With Leapp, you can upgrade your {Project} or {SmartProxy} in-place therefore it is faster but imposes a downtime on the services.
```

I know, this line was untouched, but the missing dash jumped and me, and now I can't unsee it ;)